src/types.ts

```diff
@@ -24,10 +24,10 @@ export interface PublicConfiguration<
   revalidateWhenStale: boolean
   shouldRetryOnError: boolean
   suspense?: boolean
-  initialData?: Data
+  fallbackData?: Data
   fetcher?: Fn
   use?: Middleware[]
-  fallbackValues: { [key: string]: any }
+  fallback: { [key: string]: any }
```

Comment:
```suggestion
  fallback?: { [key: string]: any }
```

Reply:
It is always provided in the default config (as `{}`), so marked as always existing here to make hooks use it confidently. For public config, we have `Partial<>` so they're all optional.